### PR TITLE
fix imports of TooltipContent and TooltipContent

### LINKED ISSUE #1826: [Bug] Compilation error with Angular 13
```
Error: node_modules/@editorjs/editorjs/types/api/tooltip.d.ts:4:9 - error TS2614: Module '"codex-tooltip"' has no exported member 'TooltipContent'. Did you mean to use 'import TooltipContent from "codex-tooltip"' instead?

Error: node_modules/@editorjs/editorjs/types/api/tooltip.d.ts:4:25 - error TS2614: Module '"codex-tooltip"' has no exported member 'TooltipOptions'. Did you mean to use 'import TooltipOptions from "codex-tooltip"' instead?
```

Steps to reproduce:
1. ```import EditorJS from '@editorjs/editorjs';```
2. ```this._editor = new EditorJS({holder: this.id});```

Editor.js version: 2.22.2

Angular version: 13.0.2

Plugins you use with their versions: none

### PR DESCRIPTION
**Pull request by raihaniabd**
Resolves #1826

### CODE REVIEW

**Comment by solovevnikita** [comment]
Any ETA for this fix?

File: types/api/tooltip.d.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
 /**
  * Tooltip API
  */
-import {TooltipContent, TooltipOptions} from 'codex-tooltip';
+import TooltipOptions from 'codex-tooltip';
+import TooltipContent from 'codex-tooltip';
```
Comment (neSpecc) [inline types/api/tooltip.d.ts:5]: Looks strange. Please, describe what you've discovered

**Review by prashant-sirohi — APPROVED** [review]
worked well

**Comment by liaozhangpeng** [comment]
@khaydarov @gohabereg  could we have a quick review on this pr. pls

File: types/api/tooltip.d.ts (hunk 1 above)
Comment (iDGE90) [reply types/api/tooltip.d.ts:5]: codex-tooltip

`TooltipOptions` and `TooltipContent` are only imported in `node_modules/codex-tooltip/dist/src/index`.
They need to be exported too to be accessible?
Which will make this PR obsolete.

**Comment by BrentonScott** [comment]
Any update on this? Still find the same error on 2.23.1

**Comment by liaozhangpeng** [comment]
> 

codex-tooltip version 1.0.2 will temporarily solve this problem

add this to your package.json

  "resolutions": {
    "codex-tooltip": "1.0.2"
  }

then
yarn or npm install

**Comment by constantlb** [comment]
Please guys, let's merge it

File: types/api/tooltip.d.ts (hunk 1 above)
Comment (PascalPiche) [reply types/api/tooltip.d.ts:5]: fix angular build. PR-1966 is a clone also

**Comment by raihaniabd** [comment]
**Temporary solution if you are using docker**

Create the file `tooltip.d.ts` with the fix, then add this to your Dockerfile

`RUN cat /app/src/app/tmp/tooltip.d.ts > /app/node_modules/@editorjs/editorjs/types/api/tooltip.d.ts`

**Comment by SzilardVigh** [comment]
Do you have any update on this fix?
@khaydarov @gohabereg

**Comment by GeekaN2** [comment]
Fixed in https://github.com/codex-team/editor.js/pull/1999

**Pull request closed without merging**